client/src/utils.js

```diff
@@ -13,8 +13,8 @@ export const colors = ['purple', 'gold', 'cyan', 'blue', 'pink', 'yellow', 'gree
 export const isServer = window === '__server';
 export const history = isServer ? createMemoryHistory() : createBrowserHistory();
 
-export const REVERSIM_SUMMIT = 'Reversim Summit 2018';
-export const REVERSIM_MAIL = 'rs18-team@googlegroups.com';
+export const REVERSIM_SUMMIT = 'Reversim Summit 2019';
+export const REVERSIM_MAIL = 'rs19-team@googlegroups.com';
```

Comment:
Actually it's `rs19team@googlegroups.com`

Reply:
@rantav are you familiar with "suggested changes"? https://twitter.com/github/status/1053360435074949120

Reply:
Fix: 

```suggestion
export const REVERSIM_MAIL = 'rs19team@googlegroups.com';
```

there I used it ;-)